doc/building.md

```diff
@@ -87,13 +87,23 @@ if all packages can be installed.
 
 ### Prerequisite steps for Mac OS X users (OS X 10.10 Yosemite)
 
- - Install [homebrew](http://brew.sh). Other package managers and building from source might also work, but aren't supported as of now. If you figure out fixes for those other options, you're welcome to open pullrequests, as long as the homebrew build doesn't break.
- - Install the needed fonts (e.g. using [homebrew-cask](https://github.com/caskroom/homebrew-cask)).
+ - Install [Homebrew](http://brew.sh). Other package managers and building from source might also work, but aren't supported as of now. If you figure out fixes for those other options, you're welcome to open pullrequests, as long as the homebrew build doesn't break.
+ - `brew update`
+ - `brew install caskroom/cask/brew-cask`
+ - `brew tap caskroom/fonts`
+ - `brew tap homebrew/version`
  - `brew tap homebrew/python`
  - `brew update`
```

Comment:
True, but shouldn't it be updated before installing `cask`, and then after the `taps`?